.swift-version

```diff
@@ -0,0 +1 @@
+5.0-DEVELOPMENT-SNAPSHOT-2019-02-06-a
```

Comment:
Can we remove this and add `.swift-version` to gitignore? That way if you use switenv locally, you can create the file. We used to commit these in the past, and they end up getting outdated and causing problems.